infra/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "infra",
+  "version": "1.0.0",
+  "description": "This is kubernetes infrastructure",
+  "main": "index.js",
+  "scripts": {
+    "infra:deploy": "kubectl apply -k $PROJECT_NAME -n=$NAME_SPACE",
+    "infra:change:docker:image": "kubectl set image deployments/$PROJECT_NAME $PROJECT_NAME=$DOCKER_IMAGE -n=$NAME_SPACE",
+    "infra:apply:configmap": "kubectl apply -f $PROJECT_NAME/configMap.yaml -n=$NAME_SPACE",
+    "infra:create:namespace": "kubectl create namespace $NAME_SPACE",
+    "infra:update:configmap": "kubectl patch configmap $CONFIG_MAP_NAME --type=merge -p \"{\\\"data\\\": {\\\"$VARIABLE_NAME\\\": \\\"$VALUE\\\"}}\" -n=$NAME_SPACE"
```

Comment:
in theory, I can run `npm run infra:update:configmap --prefix=infra/` and this will patch configmap without any extra sanity checks. And this can ruin the whole app :) 
I'd suggest to extract this command to separate bash script and do sanity check there - make sure that respective env variables are set